lib/panoptes/client/version.rb

```diff
@@ -1,5 +1,5 @@
 module Panoptes
   class Client
-    VERSION = "0.3.6".freeze
+    VERSION = "0.3.7".freeze
```

Comment:
I get it, but generally speaking bumping a version in a PR is frowned upon.

Reply:
I'm confused--how else do we version the gem?